README.md

```diff
@@ -109,7 +109,7 @@ The `POSTGRES_` variables won't be used in development mode (`NODE_ENV=developme
 | `WALLET_NAME`             | The name of the wallet to use.                                                                                                                                                                                                                                                    |
 | `WALLET_KEY`              | The key to unlock the wallet.                                                                                                                                                                                                                                                     |
 | `INVITATION_URL`          | Optional URL that can be used as the base for the invitation url. This would allow you to render a certain web page that can extract the invitation form the `oob` parameter, and show the QR code, or show useful information to the end-user. Less applicable to mediator URLs. |
-| `POSTGRES_DATABASE_URL`   | The postgres database url.                                                                                                                                                                                                                                                        |
+| `POSTGRES_DATABASE_URL`   | The postgres database url. Should be host:port (e.g. `10.10.10.10:3600`)                                                                                                                                                                                                                                                       |
```

Comment:
If we always have a port + url, it might be good to use seperate env vars to make it easier.

Reply:
That makes sense. For now I'd like to keep it like this, as otherwise I have to change code, republish the docker images etc...